hazelcast/src/test/java/com/hazelcast/topic/TopicStressTest.java

```diff
@@ -51,11 +52,16 @@
     private CountDownLatch startLatch;
     private PublishThread[] publishThreads;
     private Map<String, List<MessageListenerImpl>> listenerMap;
+    
+    @Parameterized.Parameter
```

Comment:
apparently the `HazelcastSerialClassRunner` supports parametrization thanks to #7720.